### ISSUE TITLE
Howtos should have next/prev/up links

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11559]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#246|
|Type|enhancement|
|Created|2003-08-31 05:02:03Z|

Since [#4500](https://github.com/twisted/twisted/issues/4500) has landed, and we are now using Sphinx rather than Lore, this is effectively fixed.

<details><summary>Searchable metadata</summary>

```
trac-id__246 246
type__enhancement enhancement
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation_EndOfLore documentation EndOfLore
time__1062306123000000 1062306123000000
changetime__1468044991813942 1468044991813942
version__ 
owner__khorn khorn
cc__spiv cc__moshez cc__thijs cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
I'll take this one.  It's been on my mental todo list for a
while.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

This should be fixed if/when we switch to Sphinx. See http://twistedsphinx.funsize.net/ for an example.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set status to `closed`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 20df2cf556dce3818e7e71047cefea5b4aac4862
```
#!CommitTicketReference repository="" revision="20df2cf556dce3818e7e71047cefea5b4aac4862"
Merge pull request #246 from twisted/8522-rodrigc-file

Author: rodrigc
Reviewer: Lukasa
Fixes: #8522
Replace file() with open() in twisted/scripts
```
